integration/aggregate_stats_compat_test.go

```diff
@@ -81,6 +81,21 @@ func TestAggregateCompatCollStats(t *testing.T) {
 				targetCollection := targetCollections[i]
 				compatCollection := compatCollections[i]
 				t.Run(targetCollection.Name(), func(t *testing.T) {
+					require.Equal(t, compatCollection.Name(), targetCollection.Name())
+
+					// call validate to updated statistics
+					command := bson.D{{"validate", targetCollection.Name()}}
+					targetErr := targetCollection.Database().RunCommand(ctx, command).Err()
+					compatErr := compatCollection.Database().RunCommand(ctx, command).Err()
+
+					if targetErr != nil {
```

Comment:
Is it possible in this test?

Reply:
For non existent collection added by `AddNonExistentCollection` flag, validate fails for both compat and target.